api-catalog-ui/frontend/package.json

```diff
@@ -14,7 +14,7 @@
         "exception-formatter": "1.0.7",
         "history": "4.7.2",
         "install": "0.12.2",
-        "lodash": "4.17.20",
+        "lodash": "^4.17.21",
```

Comment:
Use a specific version again, please.

Reply:
Yeah, it was automatically set by npm audit, I will change it